package.json

```diff
@@ -62,10 +62,9 @@
     "request": "^2.74.0",
     "screwdriver-config-parser": "^2.0.0",
     "screwdriver-data-schema": "^12.0.0",
-    "screwdriver-datastore-dynamodb": "^2.1.0",
-    "screwdriver-datastore-imdb": "^1.0.0",
```

Comment:
They can't use the imdb plugin in. At least until we fix the imdb to be correct.